msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,15 +176,138 @@ The Microsoft Teams JavaScript client SDK is a rich SDK used to create Tabs usin
+* The cache state is monitored every 12 seconds and the apps that don’t meet the requirements are removed from the cache.
+
+> [!NOTE]
+> Based on the configuration updates, the parameters can be modified.
+
+### Code example
+
+The following code snippet is an example of `teamsCore.registerBeforeUnloadHandler` and `teamsCore.registerOnLoadHandler` APIs:
+
+```javascript
+teamsCore.registerBeforeUnloadHandler((readyToUnload) => { 
+console.log("got beforeunload from TEAMS"); 
+// dispose resources and then invoke readyToUnload 
+readyToUnload(); 
+return true; 
+}); 
+
+teamsCore.registerOnLoadHandler((data) => { 
+console.log("got load from TEAMS", data.contentUrl, data.entityId); 
+// use contentUrl to route to correct page 
+// invoke notifySuccess when ready  
+microsoftTeams.app.notifySuccess(); 
+}); 
+```
+
+### Limitations
+
+The following are the limitations for app caching:
+
+* Single-page apps that use client-side routing for page navigation can benefit from app caching. It's recommended that the same domain is used across all contexts of your app launch.
+
+* Apps need to re-register for events such as `themeChange`, `focusEnter`, and so on, in the load handler. Teams client won't send any notifications to the app when cached. If your app requires notifications even when cached, caching might not be the right solution.
+
+* App caching is supported only in Teams desktop client. In Teams web client, even if the app registers load handlers, the app is removed from the cache after the unload sequence is completed.
+
+* Register the `load` and `beforeUnload` handlers early in your launch sequence. If the Teams client doesn’t identify these registrations before the user goes out of the app, the app isn't cached.
+
+* The Teams client invokes the `loadHandler` only after the `unload` sequence of the app is completed. For example, if a user launches tab A of your app and then launches tab B of the same app, tab B won't get the load signal until the tab A invokes the `readyToUnload` callback.
+
+* After an app moves to the cached state, it has a one-minute grace period to get the memory usage under the allowed threshold of 225 MB. If the app exceeds the threshold for more than a minute, the app is removed from the cache. The memory usage value of the app is the *workingSetSize* of the webview in the Electron [getappMetrics API](https://www.electronjs.org/docs/latest/api/app#appgetappmetrics).
+
+* Apps are cached on a per-window basis.
+
+* App caching isn't supported for the meeting stage or Task module contexts, because these can be opened on top of the tab and the same webview can't be used to render the content in the tab and the Task module.
+
+* App caching happens on a per app (not on a per tab) basis within the same window.
+
+* Apps are expected to sleep when cached (use minimal compute or network resources and minimizes SDK requests). All the register handlers and the following SDK requests are allowed when the app is cached:
+
```

Comment:
here, I would omit the `app.x`, it's confusing and this is just a list of the high level requests

Reply:
* applies to this entire list

Reply:
Removed app.x